Dockerfile.downstream

```
@@ -1,27 +1,30 @@
+ARG TARGETARCH
+
 # Build the manager binary
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23 as builder
+FROM --platform=linux/$TARGETARCH brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23 as builder
 
+ARG TARGETARCH
```

```suggestion
ARG TARGETARCH=amd64
```

We should default as same as operator & flp
others repos are not btw !